src/assets/Azure.Core.Shared/AppContextSwitchHelper.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I do not know why there is this difference (I did not even see what this difference is), but if we keep this script, and we do not need to care about this change

Reply:
Maybe you set core.autocrlf to false so that the eol difference between your file and azure.core file shows up?